.github/workflows/build-vllm-wheel.yml

```diff
@@ -169,7 +177,12 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ 'manylinux_2_28_x86_64', 'manylinux_2_28_aarch64' ]
-        device: [ 'cu128', 'cu129' ]
+        device: [ 'cu128', 'cu129', 'cu130' ]
```

Comment:
Do we really care about `cu128` here?

Reply:
Not really I think, this is just to be in sync with PyTorch.  I will clean 12.8 up later once 2.9 is out and vLLM is updated to 2.9 + CUDA 12.9 officially